iOS/CFC_Tracker/.gitignore

```diff
@@ -0,0 +1,25 @@
+build/
+CFC_Tracker/connect.plist
+*.pbxuser
+!default.pbxuser
+*.mode1v3
```

Comment:
what are these files and who generates them? I don't see them in my build directory. Why do we have to include them in the .gitignore?